ClamavSettingsForm.inc.php

```diff
@@ -66,7 +68,7 @@ function initData($request) {
 		$this->setData('pluginStylesheetURL', $basePluginUrl . 'css' . DIRECTORY_SEPARATOR);
 		$this->setData('pluginLoadingImageURL', $basePluginUrl . 'images' . DIRECTORY_SEPARATOR . "spinner.gif");
 		$this->setData('pluginLoadingImageURL', $basePluginUrl . 'images' . DIRECTORY_SEPARATOR . "spinner.gif");
-		$this->setData('pluginAjaxUrl', $baseIndexUrl . DIRECTORY_SEPARATOR . 'clamav' . DIRECTORY_SEPARATOR . 'clamavVersion');
+		$this->setData('pluginAjaxUrl', $pgRouter->url($request, null, 'clamav', 'clamavVersion'));
```

Comment:
You've removed the only usage of `$baseIndexUrl` remove the assignment on 56 as well.

Reply:
Resolved via b302a44.